lightning/backend/local.go

```diff
@@ -154,6 +168,21 @@ func (e *LocalFile) getSizeProperties() (*sizeProperties, error) {
 	return sizeProps, nil
 }
 
+func (e *LocalFile) isLocked() bool {
```

Comment:
This function is for disk-quota usage right? But I think disk-quota need not to check  isLocked, but isLockedForImport, since if it's locked but not during importing, disk quota should still wait the lock and do import after available. @kennytm  am I right?

Reply:
Let's keep this like this for now, we can add `isLockedForImport` in the disk quota PR.

Reply:
Yes. It is only for disk-quota usage

Reply:
I will remove this method and you can add what you want.